qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -192,6 +192,31 @@ class QasmSimulator(AerBackend):
       OpenMP parallelization. If parallel circuit or shot execution
       is enabled this will only use unallocated CPU cores up to
       max_parallel_threads (Default: 100).
+
+    These backend options only apply when using the ``"matrix_product_state"``
+    simulation method:
+
+    * ``"approximation_type"`` (int): Can be one of the following options:
+      NO_APPROX - means no approximation.
+      ABSOLUTE_APPROX - only the largest max_num_coefficients_for_approx
+      Schmidt coefficients are retained.
+      RELATIVE_APPROX - if the number of Schmidt coefficients is larger than
+      max_num_coefficients_for_approx, then the smallest values are
+      discarded (see approx_thresold).
+      (Default: NONE).
+
+    * ``"max_num_coefficients_for_approx"`` (int): Sets a limit on the number of Schmidt
```

Comment:
Can you call this `"matrix_product_state_max_bond_dimension"` (see comment below in controller.hpp)